Source/Core/Core/IOS/MIOS.cpp

```diff
@@ -37,7 +37,7 @@ static void ReinitHardware()
   // HACK However, resetting DI will reset the DTK config, which is set by the system menu
   // (and not by MIOS), causing games that use DTK to break.  Perhaps MIOS doesn't actually
   // reset DI fully, in such a way that the DTK config isn't cleared?
-  // DVDInterface::Reset();
+  // DVDInterface::ResetDrive(true);
```

Comment:
I'm less sure that MIOS is even supposed to reset DI in the first place.  It does make some writes to `PI_RESET_CODE`, but I haven't seen it actually write anything other than 7 (there is code that can do it, but I don't think it's live).  Of course, if it *did* write something else, that would be properly emulated because we actually run (the PPC part of) MIOS.  I don't see any writes to it from the ARM part of MIOS, nor from BC, but I might be missing them.

I also did a very basic hardware test of listening to the disc drive while launching a GameCube game; it didn't sound like it triggered a reset.